cirq/testing/consistent_act_on.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Cirq Developers
```

Comment:
this is okay to roll back given that it was created this year - sorry for my confusion

Reply:
Np. Done